core/tests/protractor_desktop/userJourneys.js

```diff
@@ -294,6 +294,20 @@ describe('Site language', function() {
     users.logout();
   });
 
+  it('should set preferred audio language selected in the Preferences page.',
+    function() {
+      users.createUser('varda@example.com', 'Varda');
+      users.login('varda@example.com');
+      preferencesPage.get();
+      preferencesPage.selectPreferredAudioLanguage('Español');
+      // TODO(DubeySandeep): Add the test to check preferred audio language
+      // choice gets reflected to the exploration player. This can be done once
+      // we will finalize a way to upload an audio file in e2e test.
+      preferencesPage.expectPreferredAudioLanguageToBe('Preferencias');
```

Comment:
Did this pass the test? If you just did preferencesPage.selectPreferredAudioLanguage, wouldn't you expect the new  PreferredAudioLanguage to be 'Español' ?

Reply:
@ctao5660, Thanks for catching this.
Fixed! :)